.htaccess

```diff
@@ -22,7 +22,6 @@ Redirect 302 /fantasy https://docs.google.com/forms/d/e/1FAIpQLSeiUE_9vJIAAo67G_
 RedirectMatch 302 (?i)/RD20 https://cmu.zoom.us/j/91610341394
 
 #top level sections
-RewriteRule		^admin[/]*$						index.php?s=admin				[NC,L]
```

Comment:
Could we redirect `/admin` to the Wordpress dashboard at `/news/wp-admin`?

Reply:
Sure